src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -27,11 +27,13 @@ class java_bytecode_convert_classt:public messaget
     symbol_tablet &_symbol_table,
     message_handlert &_message_handler,
     bool _disable_runtime_checks,
-    size_t _max_array_length):
+    size_t _max_array_length,
```

Comment:
Not intending to start a fight over words here, but "affect" is part of the English language. Nevertheless "and
we affect this kind of objects for string literals." doesn't actually tell a whole lot on what exactly is going on.